### DISCUSSION TITLE
Switching from trunk-based workflow

### DISCUSSION THREAD

**Discussion by spenserblack** [spenserblack/actions-wiki#54] [Announcements]
Right now, the README is *very* out-of-date with the initial release, and can be confusing to users. This may continue to happen in the future. For this reason, I'm considering a development branch and a release branch. The release branch would be reverted to a much earlier commit, somewhere near the initial release.

**Comment by jcbhmr** [comment]
A better way to solve this problem might be to just _release more stuff_ like v0.2.0 and v0.3.0 for each PR that is successful. That and maybe include one of those quicklinks to the v0.1.1 docs. Sorta like this -ish:

# my github wiki action

🙋‍♀️ my cool desc here

<div align=center>

![](https://placekitten.com/400/300)

[Old v0.1.1 docs](#) | [Previous releases](#) | [Awesome wikis](#)

</div>

## usage

📜 if you're looking for older version documentation, check out the [old releases](#)

---

☝ is that the kind of problem you're referring too? 🤔

**Reply by spenserblack** [reply]
Yeah.

I suppose the problem was that I was being lazy instead of making releases more quickly :laughing:
I'll make a release soon.

**Reply by jcbhmr** [reply]
If you really want to over engineer it you could make a GitHub action do it for you somehow 😜

**Reply by spenserblack** [reply]
https://github.com/spenserblack/actions-wiki/releases/tag/v0.2.0

**Comment by spenserblack** [comment]
Think I'll stick with trunk-based workflow after all. Closing.
